lib/jellyfish_web/controllers/component_controller.ex

```diff
@@ -59,7 +61,7 @@ defmodule JellyfishWeb.ComponentController do
     ]
 
   def create(conn, %{"room_id" => room_id} = params) do
-    with component_options <- Map.get(params, "options", %{}),
+    with component_options <- Map.get(params, "options", nil),
```

Comment:
If we're changing this, I'd say we should change peer_controller to behave similarly